README.adoc

```diff
@@ -192,18 +193,18 @@ Create the `InventoryManager` class in the
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=add_retry_fallback]
 ----
 
-The `@Retry` annotation dictates on what conditions we should reattempt to establish a connection
-to the `system` service and how many times it should be retried, in this case three times.
-The `@Fallback` annotation dictates which method to call when the retries are unsuccessful, in this
-case the fallbackForGet method can be used.
+The `@Retry` annotation dictates the conditions that determine if the microservice should reattempt to establish a connection
+to the `system` service and how many times it should retry. In this example, the microservice retries three times.
+The `@Fallback` annotation dictates which method to call when the retries are unsuccessful. In this
+example, you can use the `fallbackForGet` method.
```

Comment:
=>I think, "you use the `fallbackForGet` method." is good.

Reply:
What about just ,"In this example, use the `fallbackForGet` method"? (Just remove the "you.")